stdlib/src/builtin/format_int.mojo

```diff
@@ -332,8 +425,6 @@ fn _try_write_int[
         )
         writer.write(zero)
 
-        return None
```

Comment:
I think it should now be `return` since it's an early return.

Reply:
Can you explain further? there's nothing after the return, and the function itself doesn't return, so it should be unnecessary right?

Reply:
oh i notice that the original does return... i don't remember changing that, maybe it was something upstream. I'm guessing theres a reason why it returns `Optional[Error]` instead of raising then?

Reply:
I think `Optional[Error]` is like `Result[Void]`.